h/static/scripts/guest.coffee

```diff
@@ -381,8 +367,12 @@ module.exports = class Annotator.Guest extends Annotator
 
     @visibleHighlights = shouldShowHighlights
 
+  # Might not be needed anymore. Perhaps should just use on adderclick or perhaps new note?
   addComment: ->
```

Comment:
Yeah, the current form simply repeats our adderClick handle. Should be re-routed there. (Maybe setting the required action flag in the event target first.)